sentry.edge.config.ts

```diff
@@ -0,0 +1,9 @@
+import * as Sentry from '@sentry/nextjs';
+import { isSentryActivated } from '#utils/sentry';
+if (isSentryActivated) {
+  Sentry.init({
+    dsn: process.env.SENTRY_DSN,
+    tracesSampleRate: 0.1,
+    maxBreadcrumbs: 0, // dont log breadcrumb
+  });
+}
```

Comment:
Is it necessary when we dont have edge functions ?

Reply:
I agree, it shouldn't be there. There was a warning without it, but I'm going to investigate further and see if we can do without.

Reply:
Well, I just removed it, the warning is just a warning that we can safely ignore while we're not using Edge functions